src/swap/SwapScreen.tsx

```diff
@@ -54,7 +53,7 @@ import { divideByWei } from 'src/utils/formatting'
 import { getFeeCurrencyAndAmount } from 'src/viem/prepareTransactions'
 import networkConfig from 'src/web3/networkConfig'
 
-const FETCH_UPDATED_QUOTE_DEBOUNCE_TIME = 500
```

Comment:
it felt laggy to have half a second worth of wait time between the user stops typing + kicking off the quote fetch